sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# Release History
+# Release Histroy
```

Comment:
This change is incorrect. The original spelling was correct.

Reply:
Thanks. Fixed in this PR https://github.com/Azure/azure-sdk-for-net/pull/9168/files